CMakeLists.txt

```diff
@@ -85,10 +86,23 @@ ELSE()
         SET(CMAKE_LINKER_FLAGS_DEBUG "${CMAKE_LINKER_FLAGS_DEBUG} -fsanitize=address -fsanitize=undefined")
     ENDIF()
 
+    # use, i.e. don't skip the full RPATH for the build tree
+    set(CMAKE_SKIP_BUILD_RPATH FALSE)
```

Comment:
i don't think hacking the rpath is a great way to solve this.

i'd rather suggest that the user is responsible to set the $PATH/$LD_LIBRARY_PATH. we could add convenience by doing that in run_helper.sh

but as long as the install rpath remains unchanged (which i don't think it is), it would still be fine for me.

Reply:
As i understand it, the way things are handled currently are:
`RPATH` is set to `$ORIGIN`, which effectively enables loading of shared objects from relative directories. In the master branch, `bare_etiss_processor` gets built separately, and as such also receives an `RPATH` which points to the ETISS libraries.

Leaving the RPATH settings as-is in this PR would mean that `bare_etiss_processor` now also has an `RPATH` of `$ORIGIN`, which is incorrect as `bare_etiss_processor` is not placed in the same directory as the libraries.

Two options come to mind:
1) moving the `RPATH`-related changes from the top-level `CMakeLists.txt` to `src/bare_etiss_processor/CMakeLists.txt` and restoring the top-level `CMakeLists.txt`
2) restoring the top-level `CMakeLists.txt` and adding a relative `RPATH` entry to `bare_etiss_processor`, pointing to `$ORIGIN/../lib`

Reply:
alright then i'd suggest to keep this code location unchanged to retain the homogenous cross platform behavior and move the changes to the bare_etiss target only. ideally this would also keep the `$ORIGIN` entry. whether that happens through cmake scopes or target properties does not matter to me.